launcher/ui/pages/instance/StoragePage.cpp

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-3.0-only
+/*
+ *  PolyMC - Minecraft Launcher
+ *  Copyright (c) 2022 Slendi <slendi@socopon.com>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "StoragePage.h"
+#include <qmessagebox.h>
+#include <qstorageinfo.h>
```

Comment:
These are private headers that could be changed in any future Qt revision. Replace with `<QMessageBox>` and `<QStorageInfo>`